### PR TITLE
Remove plotly

### PR DESCRIPTION
**Pull request by sammacbeth**
Render plotly plots at build time with `write_image`.

Currently has the following issues:
 * Plot sizes have to specified at build time, so they will not fit the space in the page (as page width is dynamic).
 * Plots are no longer interactive.
 * SVG generation is done with electron, which is very slow for the number of SVGs we need to generate for the site.
 * The rendering doesn't seem to be play nice with the multi-process builder, so we have to build with one thread...

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -33,8 +33,6 @@
 
     <!-- jQuery -->
     <script src="{{PATHS.static}}/js/jquery.min.js"></script>
-    <!-- Plotly JS -->
-    <script src="{{PATHS.static}}/js/plotly-v1.29.3.min.js"></script>
```
Comment (remusao) [inline templates/base.html]: :heart_eyes:

File: whotracksme/import.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+mysqldump -h tracker-db.cfheundkobhl.us-east-1.rds.amazonaws.com -u trackeradmin --password='Gb5$qj4^a6JTzC5EXY#2' --skip-extended-insert --no-create-db --no-create-info --skip-add-locks trackerdb categories trackers companies tracker_domains > mysql.sql
```
Comment (remusao) [inline whotracksme/import.sh]: Not sure if we should have a password in there.

File: whotracksme/website/builder.py [hunk 3]
```diff
@@ -81,7 +81,7 @@ def on_blog_folder_change(self):
 
     def feed_event(self, event):
         futures = []
-        with concurrent.futures.ProcessPoolExecutor(max_workers=8) as executor:
+        with concurrent.futures.ThreadPoolExecutor(max_workers=1) as executor:
```
Comment (remusao) [inline whotracksme/website/builder.py]: I have a sensation of "deja vu"

File: whotracksme/website/plotting/companies.py [hunk 4]
```diff
@@ -38,8 +38,19 @@ def overview_bars(companies, highlight=2, height=None):
             )
         )
     )
-    fig = dict(data=data, layout=layout)
-    return div_output(fig)
+    # TODO: different screen sizes
```
Comment (remusao) [inline whotracksme/website/plotting/companies.py]: For this PR or another one?

File: whotracksme/website/plotting/companies.py [hunk 5]
```diff
@@ -38,8 +38,19 @@ def overview_bars(companies, highlight=2, height=None):
...
+    fig = go.Figure(data=data, layout=layout)
+    fig.write_image('_site/static/images/company_bars.svg', height=450, width=740)
```
Comment (remusao) [inline whotracksme/website/plotting/companies.py]: Is this using a headless browser? Or does plotly support dumping svg directly?

File: whotracksme/website/plotting/plots.py [hunk 6]
```diff
@@ -50,9 +53,18 @@ def profile_doughnut(values, labels, name, color_palette=False):
...
+    image_bytes = fig.to_image(format='svg', height=200, width=161)
+
+    return f'''
+    <div class="plotly-graph-div js-plotly-plot" style="height: 100%; width: 100%;">
+        <div class="plot-container plotly">
+        <div class="svg-container" style="position: relative; width: 161px; height: 200px">
+            {image_bytes.decode('utf-8')}
+        </div>
+        </div>
+    </div>
```
Comment (remusao) [inline whotracksme/website/plotting/plots.py]: This looks very similar to another snippet of code in `companies.py`. Maybe having a generic helper would make sense?

File: whotracksme/website/plotting/sankey.py [hunk 7]
```diff
@@ -34,7 +35,18 @@ def sankey_plot(sndata):
...
+
+    return f'''
+    <div class="plotly-graph-div js-plotly-plot" style="height: 400px; width: 100%;">
+        <div class="plot-container plotly">
+        <div class="svg-container" style="position: relative; width: 730px; height: 400px; margin: auto;">
+            {image_bytes.decode('utf-8')}
+        </div>
+        </div>
+    </div>
+    '''
```
Comment (remusao) [inline whotracksme/website/plotting/sankey.py]: Same remark regarding a plotly helper.

File: whotracksme/website/plotting/trackers.py [hunk 8]
```diff
@@ -151,9 +151,17 @@ def ts_trend(ts, t, percent=True):
...
+    image_bytes = fig.to_image(format='svg', height=100, width=153)
+    return f'''
+    <div class="plotly-graph-div js-plotly-plot" style="width: 153px; height: 100px;">
+        <div class="plot-container plotly">
+        <div class="svg-container" style="position: relative; width: 153px; height: 100px;">
+            {image_bytes.decode('utf-8')}
+        </div>
+        </div>
+    </div>
+    '''
```
Comment (remusao) [inline whotracksme/website/plotting/trackers.py]: Plotly helper.

**Comment by remusao** [comment]
> SVG generation is done with electron, which is very slow for the number of SVGs we need to generate for the site.

If only we had a Python library which can export plots as SVG directly (@ecnmst wink wink)

File: whotracksme/website/plotting/trackers.py (hunk 8 above)
Comment (ecnmst) [reply whotracksme/website/plotting/trackers.py]: I'd suggest the same - maybe create a `plotly_svg.html` component and reuse everywhere. Also, would we still need `plotly-graph-div`,`js-plotly-plot` classes?

**Pull request closed without merging**